CHANGELOG.md

```diff
@@ -64,7 +64,7 @@ In general we've introduced the possibility to control variants and configuratio
 - **BREAKING CHANGE:** The colors CSS variables will get provided by [`DB UI Base`](https://github.com/db-ui/base) directly, so there's nothing we could deliver by our product for this. If you were referencing those files previously, please consume them from `DB UI Base` from now on.
 - **BREAKING CHANGE:** Removed the already deprecated enterprise specific as well as the seperate transporation icon set. These have been integrated into the general DBUX iconset.
 - **BREAKING CHANGE**: All Design Tokens are now being rendered with a `db-` prefix. (see [the migration guide](docs/migrationGuide.adoc))
-- **BREAKING CHANGE**: All `primary`, `secondary`, `feedback`, `poi` and `transportation` colors got this category (`primary`, `secondary` and `transportation`) removed from their name, so e.g. `$color-secondary-red-500` became `$db-color-red-500`. (see [the migration guide](docs/migrationGuide.adoc))
+- **BREAKING CHANGE**: All `primary`, `secondary`, `feedback`, `poi` and `transportation` colors got this category (`primary`, `secondary` and `transportation`) removed from their name, so e.g. `$color-secondary-red-500` became `$db-colors-primary-enabled`. (see [the migration guide](docs/migrationGuide.adoc))
```

Comment:
these are changes to the CHANGELOG that is related to Guidelines 2.0 and not Guidelines 3.0. Most likely this changed file should get reverted in total.

Reply:
fixed